src/medkit/medkit.tsx

```diff
@@ -158,12 +182,19 @@ export const Medkit: React.FC<MedkitParams> = (
                                     <p className="h2">
                                         {totalItemsScore} / {budget} Points
                                     </p>
-                                    <p className={'medkit__basket--warning'}>
+                                    <p className={'medkit__basket--warning text-warning'}>
                                         {/* eslint-disable-next-line max-len */}
                                         {totalItemsScore >= budget ? 'Your selections are over budget' : ''}
```

Comment:
I think the equality here should be `>` rather than `>=`. 5/5 points is listed as "over budget"